nav2_bringup/bringup/launch/localization_launch.py

```diff
@@ -101,7 +101,7 @@ def generate_launch_description():
             executable='lifecycle_manager',
             name='lifecycle_manager_localization',
             output='screen',
-            parameters=[{'use_sim_time': use_sim_time},
+            parameters=[{'use_sim_time': 'false'},
```

Comment:
shouldn't this be using the variable? Seems like it might be a debug artifact.

Reply:
Yeah, this is a mistake. :+1: